pages/[type]/[id].vue

```diff
@@ -1264,7 +1264,8 @@ const members = computed(() => {
         )
       : x.is_owner
   )
-  const rest = acceptedMembers.filter((x) => x.user.id !== owner.user.id) || []
+
+  const rest = acceptedMembers.filter((x) => !owner || x.user.id !== owner.user.id) || []
```

Comment:
I think it would've been better to just skip filtering altogether if owner is falsey :p 

```suggestion
  const rest = owner ? acceptedMembers.filter((x) => x.user.id !== owner.user.id) : acceptedMembers
```